analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/LocksReleasedAllPaths.Monitor.TryCatch.cs

```diff
@@ -52,7 +52,7 @@ public void Method4(bool condition)
 
         public void Method5(string arg)
         {
-            Monitor.Enter(obj); // Compliant
+            Monitor.Enter(obj); // Noncompliant
```

Comment:
This was actually an FN. In case of a not-NullReference-Exception, `Exit` would not be called.